optaplanner-core/src/main/java/org/optaplanner/core/impl/heuristic/move/AbstractMove.java

```diff
@@ -27,7 +28,7 @@
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  * @see Move
  */
-public abstract class AbstractMove<Solution_> implements Move<Solution_> {
+public abstract class AbstractMove<Solution_> implements Move<Solution_>, Serializable {
```

Comment:
This is dangerous

Reply:
@ge0ffrey I was pretty sure someone will notice this and raise concert during PR review - thanks :) The issue findbugs has is that classes being marked as Serializable should have all the Object fields either Serializable too or transient.

More general question, do we use any classes in OptaPlanner for Java serialization? Or are we forced to declare some classes as Serializable by some framework?